code/modules/unit_tests/json_savefile_importing.dm

```diff
@@ -0,0 +1,54 @@
+/**
+ * The job of this unit test is to ensure that save files are correctly imported from BYOND to JSON
+ * Its a rather convoluted process and so this test ensures that something didnt fucking up somewhere
+ */
+/datum/unit_test/json_savefiles
+	var/savefile/test_savefile
+	var/datum/json_savefile/json_savefile
+
+	var/list/basic_list
+	var/list/assoc_list
+	var/var_string
+
+/datum/unit_test/json_savefiles/proc/setup_our_shit()
+	var/path_sfile = "data/temp.sav"
+	var/path_jfile = "data/temp.json"
```

Comment:
Full names for these rather than single letter differences (which are very hard to spot)